### PR TITLE
android acs 2.0.5 release notes + add bundled asset info

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/adobe-campaign-standard/index.md [hunk 1]
```diff
@@ -189,6 +189,25 @@ iOS
...
+
+A bunded image asset may be specified on the Campaign Standard UI to be used as a primary image or as a fallback image in the case where a specified remote image url is inaccessible. The bundled image should be specified on the Campaign Standard UI with the file name and file extension. For example, in the `Media URL` or `Bundled Image` text entry field on the Campaign Standard UI, a bundled JPEG file with the file name `adobe` can be provided in the following format:
+
+```text
+adobe.jpg
+```
+
+The specified bundled image must then be included with your app when it is built. To do so:
+
+Android
```
Comment (calebk1) [inline src/pages/documentation/adobe-campaign-standard/index.md:202]: We should H4 this.

File: src/pages/documentation/adobe-campaign-standard/index.md [hunk 2]
```diff
@@ -189,6 +189,25 @@ iOS
...
+
+1. The image must be placed in your app's `assets` directory. This directory is found in the `src/main/` directory of the app. If the directory is not present, it can be created following a  `src/main/assets` directory structure.
+
+iOS
```
Comment (calebk1) [inline src/pages/documentation/adobe-campaign-standard/index.md:206]: Same here.

File: src/pages/documentation/adobe-campaign-standard/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## July 12, 2023
+
+### Android Campaign 2.0.5
+
+- Fixed a bug where a bundled asset was not being used in a fullscreen message when the remote image url was inaccessible.
```
Comment (calebk1) [inline src/pages/documentation/adobe-campaign-standard/release-notes.md:7]: Nit, but "... remote image URL..".

File: src/pages/documentation/release-notes/index.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## July 12, 2023
+
+### Android Campaign 2.0.5
+
+- Fixed a bug where a bundled asset was not being used in a fullscreen message when the remote image url was inaccessible.
```
Comment (calebk1) [inline src/pages/documentation/release-notes/index.md:11]: Nit, but "... remote image URL..".

**Review by calebk1 — APPROVED** [review]
LGTM! Just a few very small changes.

File: src/pages/documentation/adobe-campaign-standard/index.md [hunk 5]
```diff
@@ -189,6 +189,25 @@ iOS
 
 Giving a value of `0` when setting `campaign.registrationDelay` will send a registration request on every launch event. This is the previous behavior seen before the registration request reduction enhancement was added.
 
+### Using a bundled asset as a fullscreen in-app message fallback image
+
+A bunded image asset may be specified on the Campaign Standard UI to be used as a primary image or as a fallback image in the case where a specified remote image url is inaccessible. The bundled image should be specified on the Campaign Standard UI with the file name and file extension. For example, in the `Media URL` or `Bundled Image` text entry field on the Campaign Standard UI, a bundled JPEG file with the file name `adobe` can be provided in the following format:
```
Comment (swarna04) [inline src/pages/documentation/adobe-campaign-standard/index.md]: bunded->bundled

Comment (swarna04) [reply src/pages/documentation/adobe-campaign-standard/index.md]: Let's ask the customer to provide the bundled image in the `Bundled Image` field which I suppose is meant for it.

File: src/pages/documentation/adobe-campaign-standard/index.md [hunk 6]
```diff
@@ -189,6 +189,25 @@ iOS
...
+
+```text
+adobe.jpg
+```
+
+The specified bundled image must then be included with your app when it is built. To do so:
+
+#### Android
```
Comment (swarna04) [inline src/pages/documentation/adobe-campaign-standard/index.md:202]: Should we use Tabs block here?

File: src/pages/documentation/adobe-campaign-standard/release-notes.md [hunk 7]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## July 12, 2023
+
+### Android Campaign 2.0.5
+
+- Fixed a bug where a bundled asset was not being used in a fullscreen message when the remote image URL was inaccessible.
```
Comment (swarna04) [inline src/pages/documentation/adobe-campaign-standard/release-notes.md:7]: I believe bundled image was not being picked up even when specified alone so should we omit `when the remote image URL was inaccessible.`?

```
Fixed an issue where a bundled asset was not being displayed when specified during in-app message authoring on Adobe Campaign Standard UI.
```

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: src/pages/documentation/adobe-campaign-standard/index.md (hunk 6 above)
Comment (calebk1) [reply src/pages/documentation/adobe-campaign-standard/index.md:202]: Actually yeah, a tab _would_ be a good use here. Let's do that instead.

**Comment by swarna04** [comment]
Thank you @rymorale, this section will be super helpful for the ACS customers using bundled assets in their in-app implementation.

File: src/pages/documentation/adobe-campaign-standard/index.md (hunk 6 above)
Comment (rymorale) [reply src/pages/documentation/adobe-campaign-standard/index.md:202]: sure, will update this

File: src/pages/documentation/adobe-campaign-standard/release-notes.md (hunk 7 above)
Comment (rymorale) [reply src/pages/documentation/adobe-campaign-standard/release-notes.md:7]: a bundled image alone is working. the issue fixed occurred only when a remote url was provided with a bundled image.

File: src/pages/documentation/adobe-campaign-standard/index.md [hunk 8]
```diff
@@ -189,6 +189,25 @@ iOS
...
+
+1. The image must be placed in your app's `assets` directory. This directory is found in the `src/main/` directory of the app. If the directory is not present, it can be created following a  `src/main/assets` directory structure.
+
+#### iOS
+
+1. Add the image file to your project by going to Xcode's `File > Add Files to "Your App Name"... ` menu and locating the image file that will be bundled with the app. Ensure that the targets that will be using the image file are checked in the `Add to targets` selection menu.
+2. In the `Build Phases` configuration of the app target locate the `Copy Bundle Resources` section. Click the `+` button to add the image file to be bundled with the app.
```
[the remark this answers is not in the document]
Comment (rymorale) [reply src/pages/documentation/adobe-campaign-standard/index.md:209]: you are correct, i will remove step 2.

File: src/pages/documentation/adobe-campaign-standard/tabs/index.md [hunk 9]
```diff
@@ -61,7 +61,7 @@ import AEPServices
 @import AEPServices;
 ```
 
-<Variant platform="android" task="register" repeat="4"/>
+<Variant platform="android" task="register" repeat="6"/>
```
Comment (swarna04) [inline src/pages/documentation/adobe-campaign-standard/tabs/index.md:64]: 👍

**Review by swarna04 — APPROVED** [review]

**Pull request merged**